src/test/java/core/basesyntax/service/impl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriterService;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterServiceImplTest {
+    private static FileWriterService fileWriterService;
+    private static final String DEFAULT_PATH = "src/test/resources/fruitsReportTest.csv";
+    private static final String DEFAULT_RESULT = "fruits,quantity" + System.lineSeparator()
+            + "banana,152" + System.lineSeparator()
+            + "apple,90";
+
+    @BeforeClass
+    public static void setUp() {
+        fileWriterService = new FileWriterServiceImpl();
+    }
+
+    @Test
+    public void writeToFile_isValid() {
+        fileWriterService.writeToFile(DEFAULT_RESULT, DEFAULT_PATH);
+    }
```

Comment:
to check how method worked - we should read content of this file and compare it with DEFAULT_RESULT